### DISCUSSION TITLE
Controlling font size on a slide in pptx format

### DISCUSSION THREAD

**Discussion by yonisidi** [quarto-dev/quarto-cli#10051] [Q&A]
Is there a way similar to reveal {.smaller} in pptx formats to control the font size on a specific slide in quarto?

I have text that is overflowing the templated box and I want to programmatically make the font smaller in order to get the text to fit properly.

**Comment by boshek** [comment]
Have you looked into powerpoint templates at all? I think that's the canonical way to do this: https://quarto.org/docs/presentations/powerpoint.html#powerpoint-templates

But since this is just for a specific slide, you could also try:

````qmd
::: {style="font-size: 70%;"}
## Specific Slide title
::: 
````

Does that work?

**Reply by yonisidi** [reply]
Yes I have invested a lot of time reading the documentation (it should be assumed people read documentation and need more help thus open a Q&A).

And no that does not work, since it is reveal notation and assumes a css to reduce the font.

**Reply by mcanouil** [reply]
The syntax showed will only works for HTML, as there is no "style" attribute in OpenXML or other formats.

For Word there is a Pandoc syntax, it might work with PowerPoint. If not, then you'll have to learn OpenXML and make a Lua filter.

See <https://pandoc.org/MANUAL.html#output>.

````qmd
[Get out]{custom-style="Emphatically"}, he said.

::: {custom-style="Poetry"}
A Bird came down the Walk
:::
````
